frontend/src/config/bhutan/layers.json

```diff
@@ -0,0 +1,288 @@
+{
+  "admin1_boundaries": {
+    "type": "boundary",
+    "path": "data/bhutan/btn_adm1.json",
+    "opacity": 0.8,
+    "admin_code": "Adm1_Code_",
+    "admin_level_codes": ["Adm1_Code_"],
+    "admin_level_names": ["Adm1_En"],
+    "admin_level_local_names": ["Adm1_En"],
+    "styles:": {
+      "fill": {
+        "fill-opacity": 0
+      },
+      "line": {
+        "line-color": "gray",
+        "line-width": 2,
+        "line-opacity": 0.8
+      }
+    }
+  },
+  "admin_boundaries": {
+    "type": "boundary",
+    "path": "data/bhutan/btn_adm2.json",
+    "opacity": 0.8,
+    "admin_code": "Adm2_Code_",
+    "admin_level_codes": ["Adm1_Code_", "Adm2_Code_"],
+    "admin_level_names": ["Adm1_En", "Adm2_En"],
+    "admin_level_local_names": ["Adm1_En", "Adm2_En"],
+    "styles:": {
+      "fill": {
+        "fill-opacity": 0
+      },
+      "line": {
+        "line-color": "gray",
+        "line-width": 1,
+        "line-opacity": 0.8
+      }
+    }
+  },
+  "rainfall_dekad": {
```

Comment:
Let's rely on the shared config as much as possible here.
If we are hindered by admin areas/boundaries, let's make a note of it in https://github.com/WFP-VAM/prism-app/issues/1212

Reply:
@ericboucher - I added a comment in 1212 as suggested. I could add `chart_data` to `shared/layers.json` but the levels fields will be wrong. Should we add it anyway, or just skip that element in the shared config, knowing either will break the chart feature unless the local config addresses it?

Reply:
let's keep it as you did for now I think, it's clearer